docs/user/en/reference/Resource to Shapefile Wizard.rst

```diff
@@ -1,4 +1,4 @@
-Resource to Shapefile Wizard
```

Comment:
Wonderung if it would be neccessary to rename file either. Might leed to update references as well

Reply:
see ouput of the build-log:

```
 [exec] /home/runner/work/udig-platform/udig-platform/docs/user/en/reference/Export Wizard.rst:12:toctree contains reference to nonexisting document 'reference/All Features to Shapefile Wizard'
 ```